BridgeAppSDK/SBAConsentDocumentFactory.swift

```diff
@@ -32,91 +32,12 @@
 //
 
 import ResearchKit
```

Comment:
Do we want to eventually refactor away BridgeAppSDK's direct dependency on ResearchKit, relying on ResearchUXFactory to wrap the underlying implementation details for us and allowing us to potentially "plug in" other UX implementations someday? Not a current design goal but maybe something to consider? Or maybe it doesn't even make sense. Just thinking on the screen here.